giskard/datasets/metadata/text_metadata_provider.py

```diff
@@ -19,10 +18,6 @@ def generate_metadata(self, values: pd.Series):
         return pd.DataFrame(
             {
                 "text_length": values.map(len),
-                "avg_word_length": values.map(_avg_word_length),
-                "charset": pd.Categorical(values.map(_detect_charset)),
-                "avg_whitespace": values.map(_avg_whitespace),
-                "avg_digits": values.map(_avg_digits),
```

Comment:
Let's keep this for now, I would open a separate PR. I would also keep the charset metadata initially.